test/core/slice/b64_test.cc

```diff
@@ -69,47 +72,47 @@ static void test_full_range_encode_decode_b64(int url_safe, int multiline) {
     grpc_core::ExecCtx exec_ctx;
     b64 = grpc_base64_encode(orig, sizeof(orig) - i, url_safe, multiline);
     orig_decoded = grpc_base64_decode(b64, url_safe);
-    GPR_ASSERT(GRPC_SLICE_LENGTH(orig_decoded) == (sizeof(orig) - i));
```

Comment:
This and the following line could be replaced with:

ASSERT_EQ(StringViewFromSlice(orig_decoded), string_view(orig, sizeof(orig)))

Reply:
Done.